### PR TITLE
Remove errornous and duplicate proc for time.

### PR DESCRIPTION
**Pull request by eswordthecat**

### CODE REVIEW

File: code/__HELPERS/time.dm [hunk 1]
```diff
@@ -3,9 +3,11 @@
 #define MINUTES * 600
 #define HOURS   * 36000
 
-//Returns the world time in english
-proc/worldtime2text(timestamp = world.time)
-	return "[round(timestamp / 36000) + 12]:[(timestamp / 600 % 60) < 10 ? add_zero(timestamp / 600 % 60, 1) : timestamp / 600 % 60]"
+/proc/timestamp2text(timestamp = 0, hour_to_add = 0)
```
Comment (ComicIronic) [inline code/__HELPERS/time.dm]: Shouldn't the hour to add default to 12?

Comment (eswordthecat) [inline code/__HELPERS/time.dm]: Go 4 lines more.

Comment (ComicIronic) [inline code/__HELPERS/time.dm]: Sorry, let me try again:

in order to preserve current functionality, shouldn't hour_to_add default to 12 because that's what the current code does?

Comment (eswordthecat) [inline code/__HELPERS/time.dm]: Okay lets try it again:

Go read 4 lines below this line.

Comment (eswordthecat) [inline code/__HELPERS/time.dm]: Okay you see, it's not 12 because you can pass world.timeofday there and others.
worldtime2text is the widely used function for game time so it gets the config call to hour_start
